docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -4,237 +4,205 @@ ms.date: 03/13/2020
 
 ### Extract folder
-
 Extracts an archive file into a SharePoint folder (for example, .zip).
 
 ### Get all lists and libraries
-
 Gets all lists and libraries.
 
 ### Get attachment content
-
 Returns file contents, using the file identifier. The contents can be copied somewhere else, or be used as an attachment.
 
 ### Get attachments
-
 Returns the list of attachments for the specified list item. To get to the contents of the file, add a **Get attachment content** step, and use the **File identifier** property returned by this action.
 
 ### Get file content
-
 Gets file contents using the file identifier. You can copy the contents somewhere else, or use it as an attachment.
 
 ### Get file content using path
-
 Gets file contents, using the file path.
 
 ### Get file metadata
-
 Gets info about the file, such as size, etag, created date, and so on. Uses a file identifier to select the file. To get to the values stored in the columns in the library, use the **Get file properties** action.
 
 ### Get file metadata using path
-
 Gets info about the file, such as size, etag, created date, and so on. Uses a file path to select the file. To get to the values stored in the columns in the library, use the **Get file properties** action.
 
 ### Get file properties
-
 Gets the properties saved in the columns in the library for the item specified by the item id. To get to the contents of the file, add a "Get file content" step, and use the "File identifier" property returned by this action. When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to.
 
 ### Get files (properties only)
-
-Gets the properties saved in the columns in the library for all folders and files stored in the library. You can also filter down to the items that match a condition. To work with the output from this action, use the **Apply to each** section. When using this with the On-Premises Data Gateway, you may need to be manually enter the name of the library to connect to.
+Gets the properties saved in the columns in the library for all folders and files stored in the library. You can also filter down to the items that match a condition. To work with the output from this action, use the **Apply to each** section. When using this with the On-Premises Data Gateway, you may need to be manually enter the name of the library to connect to. For more information about this action, visit the following documentation: [In-depth analysis into **Get items** and **Get files** SharePoint actions for flows in Power Automate](../power-automate/guidance/working-with-get-items-and-get-files.md).
```

Comment:
...When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to. For more info about this action, see...(remove 'be' before 'manually' as indicated in rewritten sentence. Also, rebuild 'for more info' sentence as shown above).